pkg/apis/config/scheme/scheme_test.go

```diff
@@ -86,6 +86,7 @@ profiles:
       metricProvider:
         type: Prometheus
         address: http://prometheus-k8s.monitoring.svc.cluster.local:9090
+        insecureSkipVerify: false
```

Comment:
Can you remove this and verify if it's defaulted to true?

Reply:
@Huang-Wei , we tested the default behavior in [#L81](https://github.com/kubernetes-sigs/scheduler-plugins/blob/4a78c35c49db51d64e45eea1930e5fbf39d954cd/pkg/apis/config/scheme/scheme_test.go#L81) and [#L132](https://github.com/kubernetes-sigs/scheduler-plugins/blob/4a78c35c49db51d64e45eea1930e5fbf39d954cd/pkg/apis/config/scheme/scheme_test.go#L132)

Reply:
I see. Thanks.